validators/terravaloper16f9xsgwvw93pzugeztz8njmlpcveh0cw2rg3jn/profile.json

```diff
@@ -0,0 +1,5 @@
+{
+  "contact": {
+    "email": "f3hd@gmail.com",
```

Comment:
@GDeppe Please remove the trailing comma on this line to adhere to JSON formatting.

Reply:
trailing comma removed

Reply:
![Screenshot_20231114_014741_Chrome.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/ac9e10a7-6279-4eeb-a787-256ccbd93587)

Reply:
![Screenshot_20231113_163422_Binance.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/0632a745-77f4-4f25-bdaf-8b35f6aac61f)

Reply:
- [ ] **@![Screenshot_20231113_163411_Binance.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/b04ea058-befd-4d59-9ed0-0aeb1f481610)

![share_2023-07-10_23_40_06_812.jpeg_2023-07-10_23_40_06.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/def6db24-1c3a-4bdc-af98-1c06767ee983)

![Pic_1671394501325.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/6457d204-ede7-46ce-8e1b-6a8f57fbc20d)

![Screenshot_20221115-004447_Chrome.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/befc666c-c58c-4d59-aa30-64cb5c41bcf9)

![Screenshot_20221225-024519_Chrome.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/bb4c0589-34e7-4ddf-a0a0-1c8b476c0cad)

![Screenshot_20230601_010910_Chrome.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/7a927133-8264-4b1e-8549-6402392ecb79)

![Screenshot_20231020_203809_Binance.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/23c941ed-3b5a-42b7-87ca-e5c00e6a7a65)

![Screenshot_20221212-025313_Edge.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/d5cd7c77-4428-4c39-bdd2-06b72c5c75e2)

![Screenshot_20221025-003043_Chrome.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/4baf9090-4873-40c8-a522-3d20c81d4b50)

![Screenshot_20231108_223029_Binance.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/32a790e3-6617-46ed-b262-a61f5b69e523)

![share_pic__2023-08-03_23_40_05.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/e55e894c-7118-4df0-957e-70f1de409916)

![Screenshot_20221025-001726_Chrome.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/c883b8b9-df6b-4d89-8861-bf0d31b80566)

![share_pic__2023-10-10_01_13_37.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/13a355e7-1563-4e51-8dd9-cebd8f6321db)

![Screenshot_20230923_010720_Edge.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/48fb52c3-7de1-4a33-bb24-6259f59109ff)

![Screenshot_20230926_183951_Binance.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/becea183-89f9-4db6-ba64-9a1c80b2cdd1)

![Screenshot_20221029-201037_Chrome.jpg](https://github.com/terra-money/validator-profiles/assets/117018255/106c0138-6542-4d0a-b165-673eb774c9b9)

 **